manticore/ethereum.py

```diff
@@ -631,7 +631,7 @@ def get_source_for(self, asm_offset, runtime=True):
 
     @property
     def signatures(self):
-        return dict(((b, a) for (a, b) in self._hashes.items()))
+        return dict(((b.encode(), a) for (a, b) in self._hashes.items()))
```

Comment:
if we want to ref them as bytes should probably just store them as bytes rather than decoding every item on each access to signatures.

since its a hex string, should probably just be str no? this is where python static typing stuff would be handy